scripts/eurostat/health_determinants/alcohol_consumption/process.py

```diff
@@ -0,0 +1,820 @@
+                        gender = "\ngender: dcs:" + prop
+                    elif "Education" in prop:
+                        education = "\neducationalAttainment: dcs:" + \
+                            prop.replace("Or","__")
+                        #prop.replace("EducationalAttainment","")
+                        prop.replace("Or","__")
+                    elif "Percentile" in prop:
+                        incomequin = "\nincome: ["+prop.replace("Percentile",\
+                            "").replace("IncomeOf","").replace("To"," ")+" Percentile]"
+                    elif "Urban" in prop or "SemiUrban" in prop \
+                        or "Rural" in prop:
+                        residence = "\nplaceOfResidenceClassification: dcs:" + prop
+                    elif "ForeignBorn" in prop or "Native" in prop:
+                        countryofbirth = "\nnativity: dcs:" + \
+                            prop.replace("CountryOfBirth","")
+                    elif "ForeignWithinEU28" in prop or "ForeignOutsideEU28" in prop\
+                        or "Citizen" in prop or "NotACitizen" in prop:
+                        citizenship = "\ncitizenship: dcs:"+\
+                        prop.replace("Citizenship","")
+
+            final_mcf_template += mcf_template.format(ip1=sv,
+                                                      ip2=denominator,
+                                                      ip3=healthbehavior,
+                                                      ip4=gender,
+                                                      ip5=frequenc,
+                                                      ip6=education,
+                                                      ip7=incomequin,
+                                                      ip8=residence,
+                                                      ip9=countryofbirth,
+                                                      ip10=citizenship) + "\n"
+
+        # Writing Genereated MCF to local path.
+        with open(self.mcf_file_path, 'w+', encoding='utf-8') as f_out:
+            f_out.write(final_mcf_template.rstrip('\n'))
+        # pylint: enable=R0914
+        # pylint: enable=R0912
+        # pylint: enable=R0915
+
+    def process(self):
+        """
+        This Method calls the required methods to generate
+        cleaned CSV, MCF, and TMCF file.
+        Args:
+            None
+        Returns:
+            None
+        """
+
+        final_df = pd.DataFrame(columns=['time','geo','SV','observation',\
+            'Measurement_Method'])
+        # Creating Output Directory
+        output_path = os.path.dirname(self.cleaned_csv_file_path)
+        if not os.path.exists(output_path):
+            os.mkdir(output_path)
+        sv_list = []
+
+        for file_path in self.input_files:
+            df = pd.read_csv(file_path, sep='\t', header=0)
+            file_name = file_path.split("/")[-1][:-4]
+            function_dict = {
```

Comment:
Rename function_dict to file_to_function_mapping (Refer to BMI for similar changes)

Reply:
Changed function_dict to file_to_function_mapping.